app/src/androidTest/java/org/eyeseetea/malariacare/dataSources/OkHttpClientDataSourceShould.java

```diff
@@ -0,0 +1,43 @@
+package org.eyeseetea.malariacare.dataSources;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import android.support.test.runner.AndroidJUnit4;
+import android.test.suitebuilder.annotation.SmallTest;
+
+import org.eyeseetea.malariacare.data.remote.api.OkHttpClientDataSource;
+import org.eyeseetea.malariacare.domain.entity.Credentials;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+@SmallTest
+public class OkHttpClientDataSourceShould {
+
+    private static final String DHIS_CHECK_EVENT_API =
+            "/api/events.json?program=%s&orgUnit=%s&startDate=%s&endDate=%s&skipPaging=true"
+                    + "&fields=event,orgUnit,program,dataValues";
+    private Credentials mCredentials;
+
+    @Before
+    public void loadCredentials(){
+        //// TODO: 26/06/2018  add testing credentials in github
+        mCredentials = new Credentials("https://data.psi-mis.org","", "");
```

Comment:
Please analyze if it's possible to use from testing local.properties to store and retrieve credentials. Review use of local.properties, this file is not uploaded to source control and it need be configured in Gradle files